dev-libs/atf/atf-0.21.recipe

```diff
@@ -0,0 +1,107 @@
+SUMMARY="Libraries to write tests in C, C++ and shell"
+DESCRIPTION="ATF, or Automated Testing Framework, is a collection of libraries \
+to write test programs in C, C++ and POSIX shell.
+
+The ATF libraries offer a simple API. The API is orthogonal through the various \
+bindings, allowing developers to quickly learn how to write test programs in \
+different languages.
+
+ATF-based test programs offer a consistent end-user command-line interface to \
+allow both humans and automation to run the tests.
+
+ATF-based test programs rely on an execution engine to be run and this execution \
+engine is not shipped with ATF. Kyua is the engine of choice."
+HOMEPAGE="https://github.com/jmmv/atf"
+COPYRIGHT="2007, 2008, 2009, 2010, 2011, 2012, 2014 The NetBSD Foundation, Inc
+	2011, 2012 Google Inc
+	2014 Automated Testing Framework authors"
+LICENSE="BSD (2-clause)
+	BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="https://github.com/jmmv/atf/releases/download/atf-$portVersion/atf-$portVersion.tar.gz"
+CHECKSUM_SHA256="92bc64180135eea8fe84c91c9f894e678767764f6dbc8482021d4dde09857505"
+PATCHES="atf-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion="1.0.0"
+libVersionCPlus="2.0.0"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+libVersionCPlusCompat="$libVersionCPlus compat >= ${libVersionCPlus%%.*}"
+
+PROVIDES="
+	atf$secondaryArchSuffix = $portVersion
+	cmd:atf_sh = $portVersion
+	lib:libatf_c$secondaryArchSuffix = $libVersionCompat
+	lib:libatf_c++$secondaryArchSuffix = $libVersionCPlusCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	atf${secondaryArchSuffix}_devel = $portVersion
+	devel:libatf_c$secondaryArchSuffix = $libVersionCompat
+	devel:libatf_c++$secondaryArchSuffix = $libVersionCPlusCompat
+	"
+REQUIRES_devel="
+	atf$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:autoconf
+	cmd:awk
+	cmd:g++$secondaryArchSuffix
+	cmd:gcc$secondaryArchSuffix
+	cmd:grep
+	cmd:make
+	cmd:sed
+	"
+# Avoid cyclic dep
+if [ "$recipeAction" = "TEST" ]; then
+	BUILD_PREREQUIRES+="
+		cmd:kyua
+		"
```

Comment:
I'd use this:
```
if [ "$recipeAction" = TEST ]; then
BUILD_PREREQUIRES+="
	cmd:kyua
	"
fi
```
because having this indentation will reduce the size of the diff when TEST_REQUIRES becomes available.
If/when this happens, with the suggested indentation the diff will be:
```diff
-if [ "$recipeAction" = TEST ]; then
-BUILD_PREREQUIRES+="
+TEST_REQUIRES="
 	cmd:kyua
 	"
-fi
```

Reply:
I could actually remove the entire block. It doesn't work at all